ci/builder/requirements.txt

```diff
@@ -8,7 +8,7 @@ black==22.12.0
 boto3-stubs[ec2,iam,kinesis,s3,sqs,ssm,sts]==1.26.89
 boto3==1.26.64
 click==8.1.3
-cryptography==38.0.4
+cryptography>=39.0.1
```

Comment:
Actually, could you pick a specific version? All other libs here are of a specific version, this helps make the ci-builder more deterministic in its behavior

Reply:
Yes, will do! Thanks for the note!